### PR TITLE
Add proper sage path handling.

### PR DESCRIPTION
**Pull request by strarsis**
This PR adds proper handling of sage asset paths (in development + production builds).

### CODE REVIEW

File: src/helpers.php [hunk 1]
```diff
@@ -4,18 +4,91 @@
...
+	return trailingslashit(get_svg_path()) . $icon . '.svg';
+}
+
+/**
+ * Returns a path without file extension
+ *
+ * @param  string $icon
+ * @return string
+ */
+function strip_file_ext($path)
```
Comment (hybridvision) [inline src/helpers.php]: As mentioned in [my comment here](https://github.com/Log1x/blade-svg-sage/issues/11#issuecomment-360426676), this function always seems to return the filename with a `./` at the beginning of it, which causes some non-fatal problems later with the path construction.

I'd suggest you change this function to:

```php
function strip_file_ext($path)
{
	return pathinfo($path, PATHINFO_FILENAME);
}
```

Even when you define several nested directories for your `bladesvg_image_path`, this function only ever receives the filename with no directories. As such, the code above is a bit cleaner (or maybe you don't even need this function and could call `pathinfo()` inline).

Comment (strarsis) [reply src/helpers.php]: I [fixed and inlined that part](https://github.com/strarsis/blade-svg-sage/blob/0ccb18c27adffd763099032067691459d8964a8d/src/helpers.php#L74) now.

Comment (hybridvision) [reply src/helpers.php]: Great! Thanks :)

**Comment by strarsis** [comment]
👍

**Pull request merged**